features/manage-position/YourPosition.tsx

```diff
@@ -121,37 +121,40 @@ const YourPosition = () => {
           {`${_collateral} ${_collSymbol}`}
         </Status>
         <Status>
-          <Label>Tokens outstanding: </Label>
+          <Label>Tokens created: </Label>
           {`${_tokens} ${_tokenSymbol}`}
         </Status>
         <Status>
           <Label>
-            Estimated identifier price (
+            Identifier price(
             <Link href={pricefeedUrl} target="_blank" rel="noopener noreferrer">
-              Coinbase Pro
+              Coinbase
             </Link>
             ):{" "}
           </Label>
-          {_latestPrice}
+          {Number(_latestPrice).toFixed(4)}
         </Status>
         <Status>
-          <Label>(CR) Collateralization ratio:</Label>
-          {` ${pricedCR} (${_collSymbol} / ${_tokenSymbol})`}
+          <Label>Collateralization ratio(CR):</Label>
+          {` ${pricedCR}`}
         </Status>
         <Status>
-          <Label>(GCR) Global collateralization ratio:</Label>
-          {` ${pricedGCR} (${_collSymbol} / ${_tokenSymbol})`}
+          <Label>Global collateralization ratio(GCR):</Label>
```

Comment:
Same as above

Reply:
here is an example of this string on mobile:
![image](https://user-images.githubusercontent.com/12886084/88556233-12733c80-d029-11ea-9379-62c8ffd29299.png)

Reply:
and with a EMP selected looks like this:
![image](https://user-images.githubusercontent.com/12886084/88556423-4babac80-d029-11ea-94d3-217147a6041a.png)

Reply:
In general I don't think we should be fixing mobile optimization by removing spaces. We should toggle font-size or the text in general for "small" screens instead